### PR TITLE
{lib}[GCCcore/9.3.0] CFITSIO v3.48

### PR DESCRIPTION
**Pull request by bartoldeman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
build-node.computecanada.ca - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.7.7
See https://gist.github.com/a1db594b4edf83f14676996135b47564 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2a343b01ac2bc80d6fff3ce230dd6d3a for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/7362a72c123a91cb0a3c04f039a64b6e for a full test report.

File: easybuild/easyconfigs/c/CFITSIO/CFITSIO-3.48-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+homepage = 'https://heasarc.gsfc.nasa.gov/fitsio/'
+description = """CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
+FITS (Flexible Image Transport System) data format."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/']
+sources = ['%%(namelower)s-%s.tar.gz' % version]
```
Comment (lexming) [inline easybuild/easyconfigs/c/CFITSIO/CFITSIO-3.48-GCCcore-9.3.0.eb:14]: Thank you for moving `sources` below `source_urls`. This is how it should always be. We should have a test blocking PRs that do not follow this rule :slightly_smiling_face:

**Comment by lexming** [comment]
Going in, thanks @bartoldeman !

**Pull request merged**